storage/integration_test/src/integration_test.cc

```diff
@@ -47,7 +47,8 @@ namespace firebase {
 namespace storage {
 namespace internal {
 extern bool g_retry_file_not_found_for_testing;
-}
+extern bool g_retry_failed_future_for_testing;
```

Comment:
Could we have a single flag? g_retry_all_errors_for_testing

Reply:
Done. When g_retry_all_errors_for_testing is true, IsRetryableFailure will retry anything that has a http status other than 200-299 since 2xx indicates succees.